configure.ac

```diff
@@ -0,0 +1,246 @@
+AC_INIT(py-libzfs, 1.0)
+
+AC_LANG(C)
+AC_PROG_CC
+
+define(SECTION_TITLE,
+[
+	echo >& AS_MESSAGE_FD
+	echo '   $1   ' | sed -e's/./-/g' >&AS_MESSAGE_FD
+	echo '   $1' >& AS_MESSAGE_FD
+	echo '   $1   ' | sed -e's/./-/g' >&AS_MESSAGE_FD
+])
+
+dnl works for checking define statements in c
+m4_include([m4/AX_CHECK_DEFINE.m4])
+
+
+dnl Setting appropriate CFLAGS
+CFLAGS="-nostdinc -I/usr/src/sys/cddl/compat/opensolaris/uts/common/ -I/usr/src/cddl/compat/opensolaris/include/ -I/usr/src/sys/cddl/contrib/opensolaris/uts/common/
+-I/usr/src/sys/cddl/contrib/opensolaris/uts/common/fs/zfs/ -I/usr/src/sys/cddl/compat/opensolaris/sys/ -I/usr/src/sys/cddl/compat/opensolaris/ -I/usr/src/cddl/lib/libumem 
+-I/usr/src/usr/src/cddl/compat/opensolaris/ -I/usr/src/usr/src/cddl/compat/opensolaris -I/usr/src/cddl/compat/opensolaris/lib/libumem -I/usr/src/cddl/contrib/opensolaris/lib/libzpool/common 
+-I/usr/src/sys/cddl/contrib/opensolaris/common/zfs -I/usr/src/sys/cddl/contrib/opensolaris/uts/common/sys -I/usr/src/cddl/contrib/opensolaris/head -I/usr/src/cddl/contrib/opensolaris/lib/libnvpair 
+-I/usr/src/cddl/contrib/opensolaris/lib/libuutil/common -I/usr/src/cddl/contrib/opensolaris/lib/libzfs/common -I/usr/src/cddl/contrib/opensolaris/lib/libzfs_core/common ${CFLAGS} -I/usr/include"
```

Comment:
maybe make `/usr/src` an overrideable variable. some people like to checkout their source tree in esoteric places, while others may have many different versions checked out